lib/definitions/livesync.d.ts

```diff
@@ -154,7 +154,7 @@ interface ILiveSyncService {
 	 * @param {ILiveSyncInfo} liveSyncData Describes the LiveSync operation - for which project directory is the operation and other settings.
 	 * @returns {Promise<void>}
 	 */
-	liveSync(deviceDescriptors: ILiveSyncDeviceInfo[], liveSyncData: ILiveSyncInfo): Promise<void>;
+	liveSync(deviceDescriptors: ILiveSyncDeviceInfo[], liveSyncData: ILiveSyncInfo, projectFilesConfig: IProjectFilesConfig): Promise<void>;
```

Comment:
This method is exposed publicly when CLI is used as a library, is this really required?